docs/src/main/asciidoc/spring-data-support.adoc

```diff
@@ -91,7 +117,162 @@ spring:
         database: ${AZURE_COSMOS_DATABASE}
 ----
 
-=== Samples
 
-Please refer to link:https://github.com/Azure-Samples/azure-spring-boot-samples/tree/spring-cloud-azure_4.0[azure-spring-boot-samples] for more details.
+(Optional) Property `spring.cloud.azure.cosmos.consistency-level` is also supported.
+----
+spring:
+  cloud:
+    azure:
+      cosmos:
+        consistency-level: ${CONSISTENCY_LEVEL}
+----
+
```

Comment:
If we already moved this section to the configuration, should we also delete it from here?